src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -1,42 +1,73 @@
 package cromwell.engine
 
+import java.util.UUID
+
 import akka.actor.{Actor, Props}
 import akka.event.{Logging, LoggingReceive}
 import cromwell.binding.WdlExpression.ScopedLookupFunction
 import cromwell.binding.{Call, CallInputs, WorkflowDescriptor}
-import cromwell.engine.backend.Backend
+import cromwell.engine.backend.{TaskAbortedException, Backend}
 
 import scala.language.postfixOps
-import scala.util.{Failure, Success}
-
+import scala.util.{Success, Failure}
 
 object CallExecutionActor {
-  def props(backend: Backend, command: String, workflowDescriptor: WorkflowDescriptor, call: Call, callInputs: CallInputs, lookup: ScopedLookupFunction): Props =
-    Props(new CallExecutionActor(backend, command, workflowDescriptor, call, callInputs, lookup))
+  type CallExecutionID = UUID
+  type WorkflowID = UUID
```

Comment:
Same comment about `type` vs `case class`. Also doing this here would clean up all sorts of stuff (tuple boilerplate, etc)